quickstarts/aws/aws-apprunner/config.yml

```
@@ -0,0 +1,25 @@
+slug: aws-apprunner
+description: |
+  ## Monitor your Apps running in App Runner coutainers on New Rerilc Instant Observability Platform!
+  With New Relic's AWS App Runner integration, you now can see all your App Runner data on New Rerilc Instant Observability Platform.
+  ## Get started!
+  Check out the [documentation](https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-apprunner) to learn more about New 
+Relic monitoring and integration with AWS App Runner.
+summary: |
+  Use New Relic - AWS App Runner Integration to Monitor Your Conteinerized Apps and Services with New Relic
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+title: Amazon App Runner with New Relic
```

```suggestion
title: Amazon App Runner
```

These are all with New Relic, for consistency I would just leave the title `Amazon App Runner`